llama_index/llms/konko.py

```diff
@@ -222,52 +257,28 @@ def _stream_chat(
 
         def gen() -> ChatResponseGen:
             content = ""
-            function_call: Optional[dict] = None
             for response in completion_with_retry(
                 is_chat_model=self._is_chat_model,
                 max_retries=self.max_retries,
                 messages=message_dicts,
                 stream=True,
                 **all_kwargs,
             ):
-                if len(response["choices"]) == 0 and response.get("prompt_annotations"):
-                    # When asking a stream response from the Azure OpenAI API
-                    # you first get an empty message with the content filtering
-                    # results. Ignore this message
+                if len(response.choices) == 0 and response.prompt_annotations:
```

Comment:
curious whether we need to check the openai version here as well.

Reply:
Done